release.yaml

```diff
@@ -173,11 +173,14 @@ docker-images:
         name: haproxy
         repository: https://github.com/docker-library/haproxy
         tag: 2.4-alpine
+      promtail:
+        name: grafana/promtail
+        tag: 2.7.3
 helm-charts:
   logging:
-    loki-stack:
-      repository: https://grafana.github.io/helm-charts
-      version: 2.8.9
+    loki:
+      repository: https://grafana.github.io/loki/charts
+      version: 2.8.0
```

Comment:
In https://github.com/metal-stack/metal-roles/pull/150 we use the same chart repository URL and version `4.6.1`. Is this change intended or is it the same?

Reply:
loki charts are deprecated!